src/v2/DevTools/renderUntil.tsx

```diff
@@ -16,15 +16,15 @@ function renderUntil<
       if (predicate(this)) {
         resolve(this)
       } else {
-        setImmediate(() => {
+        setTimeout(() => {
```

Comment:
These were causing browser errors as `setImmediate` is a node-only function and Webpack no longer auto polyfills.

Reply:
After further testing, this one should not be required.